### PR TITLE
[QACI-628] Stabilize system prop test

### PR DESCRIPTION
**Pull request by lprimak**
Stabilized often-failing SystemPropertyTest in Payara-Smples

### CODE REVIEW

**Comment by lprimak** [comment]
jenkins test

File: appserver/tests/payara-samples/samples/rest-management/src/test/java/fish/payara/samples/rest/management/CreateInstanceTest.java [hunk 1]
```diff
@@ -37,42 +37,57 @@
...
-
-    public void afterClassEvent(@Observes AfterClass event) {
-        if (instance.isCreated()) {
-            instance.destroy();
+    String firstWord(String line) {
+        if (line == null) {
+            return line;
+        }
+        String[] words = line.split(" ");
+        if (words.length > 0) {
```
Comment (Pandrex247) [inline appserver/tests/payara-samples/samples/rest-management/src/test/java/fish/payara/samples/rest/management/CreateInstanceTest.java:87]: Shouldn't this be > 1?
It's not _wrong_ per se, but splitting a line with no spaces will return an array with one element, so surely this always passes as long as line isn't null?

Comment (lprimak) [reply appserver/tests/payara-samples/samples/rest-management/src/test/java/fish/payara/samples/rest/management/CreateInstanceTest.java:87]: you are right :) useless code and will be removed

**Comment by lprimak** [comment]
jenkins test

**Comment by lprimak** [comment]
jenkins test

**Review by cubastanley — APPROVED** [review]

**Pull request merged**